### PR TITLE
adding api document to ejbca

### PR DESCRIPTION
**Pull request by muhamedcpqd**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Adds api docs for ejbca

* **What is the current behavior?** (You can also link to an open issue here)
No api docs

* **What is the new behavior (if this is a feature change)?**
New api doc

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
no

* **Other information**:

### CODE REVIEW

File: ejbca/api-docs/apiary.apib [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+        {
+          "status": 200,
+          "message": "ok"
+        }
+
+## CRL [/ca/{caname}/crl]
+
+### Create or update current CRL [PUT]
+
++ Parameters
```
Comment (mprevide) [inline ejbca/api-docs/apiary.apib]: Tem request esse endpoint? Senao pq ele é PUT? Tem a versão com a guia que o Rafael criou?

File: ejbca/api-docs/apiary.apib [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+
+# Group EJBCA user management
+## Create or edit user [/user]
+### Create or edit a user [POST]
+
++ Request (application/json)
+  + Body
+
+            {
+                "caName": "PROTOCOL",
```
Comment (mprevide) [inline ejbca/api-docs/apiary.apib]: Todos os campos do request são obrigatórios?

File: ejbca/api-docs/apiary.apib [hunk 3]
```diff
@@ -0,0 +1,486 @@
...
+              }
+          ]
+        }
+
+
+### Revoke all cerificates from a user [DELETE /user/{username}?{reason,delete}]
+
++ Parameters
+  + username: IOTmidCA (required, string): user username
+  + reason: UNSPECIFIED (optional, enum) - one of REVOKATION_REASON enum values. default: UNSPECIFIED
```
Comment (mprevide) [inline ejbca/api-docs/apiary.apib]: Poderia ter a lista do REVOKATION_REASON

File: ejbca/api-docs/apiary.apib [hunk 4]
```diff
@@ -0,0 +1,486 @@
...
+        }
+
+## Sign a certificate for a user [/user/{username}/pkcs10]
+### Return the signed certificate [POST]
+
++ Parameters
+  + username: IOTmidCA (required, string): user username
+
++ Request (application/json)
+  In a csv file, generated with the command 'openssl req -new -out server.csr -key server.key'
```
Comment (mprevide) [inline ejbca/api-docs/apiary.apib]: Será q nao seria um csr ao inves de csv?

File: ejbca/api-docs/apiary.apib (hunk 1 above)
Comment (muhamedcpqd) [reply ejbca/api-docs/apiary.apib]: sim, a crl pode ser atualizada ou criada.

**Pull request merged**